swap/src/bin/swap.rs

```diff
@@ -428,12 +431,15 @@ where
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use crate::determine_btc_to_swap;
-    use ::bitcoin::Amount;
     use std::sync::Mutex;
+
+    use ::bitcoin::Amount;
     use tracing::subscriber;
 
+    use crate::determine_btc_to_swap;
+
+    use super::*;
```

Comment:
Same here. Are you using "organize imports" from CLion? I saw that causing this at some point.